examples/cra-kitchen-sink/src/storyshots.test.js

```diff
@@ -1,17 +1,9 @@
-import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
+import initStoryshots from '@storybook/addon-storyshots';
 import path from 'path';
-
-function createNodeMock(element) {
-  if (element.type === 'div') {
-    return { scrollWidth: 123 };
-  }
-  return null;
-}
+import { mount as renderer } from 'enzyme';
 
 initStoryshots({
   framework: 'react',
   configPath: path.join(__dirname, '..', '.storybook'),
-  test: multiSnapshotWithOptions({
-    createNodeMock,
-  }),
+  renderer,
```

Comment:
Can we support custom renderers for multisnapshots as well?

Reply:
yeah sure. done

Reply:
Looks like it still stores all the snapshots in one file

Reply:
it does. i'm currently debugging it. looks like it did not work in the first place.  the multisnapshot function doesnt seem to be called correctly on the `release/3.3` branch

Reply:
should work now. we cant use enzymes `mount` since that renders out the context prop (includes the file path, which is different on every machine). We also need to pass a serializer since jest-specific-snapshots doesnt use the projects serializer